src/main/java/com/yo1000/sonarqube/collaboration/Application.java

```diff
@@ -5,6 +5,14 @@
  */
 public class Application {
     public static void main(String[] args) {
+        if (true) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Change this condition so that it does not always evaluate to "true" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://dev.yo1000.com/coding_rules#rule_key=squid%3AS2583)